src/ensembl/src/shared/components/loading-button/LoadingButton.scss

```diff
@@ -0,0 +1,79 @@
+@import 'src/styles/common';
+
+.buttonWrapper {
+  position: relative;
+  display: inline-block;
+}
+
+.invisible {
+  visibility: hidden;
+}
+
+.loadingIndicator,
+.successIndicator,
+.errorIndicator {
+  position: absolute;
+  left: 0;
+  top: 0;
+
+  bottom: 0;
+  right: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1;
+}
+
+.spinner {
+  width: 20px;
+  height: 20px;
+  border-width: 2px;
+}
+
+.checkmark {
+  fill: $green;
+  height: 80%;
+  animation: checkmark-pulse 0.5s ease-in;
+}
+
+.cross {
+  fill: $red;
+  height: 55%;
+}
+
+.loadingIndicator,
+.successIndicator,
+.errorIndicator {
+  border-radius: 4px;
+  border: 1px solid $grey;
+}
+
+// .loadingIndicator {
+//   border: 1px solid $grey;
+// }
+
+// .successIndicator {
+//   border: 1px solid $grey;
+// }
+
+// .errorIndicator {
+//   border: 1px solid $red;
+// }
```

Comment:
Can these be removed?

Reply:
Ah, the leftover from those design tweaks 😅 Removed.